simulation/character_builder.py

```diff
@@ -164,7 +164,7 @@ def spend_xp(self, amount):
   def take_advantage(self, advantage):
     cost = Advantage(advantage).cost()
     self.spend_xp(cost)
-    self.character.take_advantage(advantage)
+    self.character().take_advantage(advantage)
```

Comment:
Everywhere else you end up saying ``self.character()`` so I changed that here.

FWIW, the idiomatic way to do this in Python would be to make ``self.character`` a property that returns ``self._character``, i.e.

```python
  @property
  def character(self):
    return self._character
```

The ``@property`` disallows assignment by default, with the ability to extend this in various ways and get all of the benefits of a function call without having to specify the ``()`` on every invocation.  What you've done works just as well, but would probably be confusing to people who aren't familiar with the Java builder pattern you're using here.  (Since I've done a lot of Java, I wasn't confused at all.)